src/test/java/org/testeditor/fixture/web/drivermanager/InternetExplorerTest.java

```diff
@@ -0,0 +1,127 @@
+package org.testeditor.fixture.web.drivermanager;
+
+import io.github.bonigarcia.wdm.BrowserManager;
+import io.github.bonigarcia.wdm.InternetExplorerDriverManager;
+import java.io.IOException;
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class InternetExplorerTest {
+
+    private WebDriver driver;
+
+    private static String proxyHost;
+    private static String proxyUser;
+    private static String proxyPassword;
+    private static String testSiteUrl;
+    private static String searchfieldID;
+    private static String searchString;
+    private static String firstSiteTitle;
+    private static String xPathResult;
+    private static String resultSiteTitle;
+    private static String ieFlakinessCapability;
+
+    @BeforeClass
+    public static void setupClass() throws IOException {
+        Assume.assumeTrue("This is not a Windows OS - ignoring test", SystemUtils.IS_OS_WINDOWS);
```

Comment:
Helpertool provides method to check? If not used, why should the helper tool provide this method?